### DISCUSSION TITLE
希望大佬可以给出一个BOM表，方便采购零件

### DISCUSSION THREAD

**Discussion by xiehd97** [peng-zhihui/ElectronBot#53] [General]
对单片机这块不怎么熟，具体型号看的不是太明白

**Comment by eqio** [comment]
电子零件芯片的BOM在另一个帖子中有。舵机、屏幕、摄像头（摄像头连接作者分享了），不在里面

**Comment by DylanZin** [comment]
我大概整理了一下，可以进群看 983344575

**Reply by tolerious** [reply]
群满了。

**Comment by Jzacky** [comment]
[electronbot_bom_all.xlsx](https://github.com/peng-zhihui/ElectronBot/files/8324276/electronbot_bom_all.xlsx)

**Reply by Vapeti3** [reply]
嘿，感謝您提供此 BOM 文件，但是您還有印刷電路板的 CPL 文件嗎？

**Comment by Kitem** [comment]
您好，我是焕挺，您的邮件已经收到，我会尽快回复！
